code/__DEFINES/nuclear_bomb.dm

```diff
@@ -26,3 +26,5 @@
 #define DETONATION_HIT_SYNDIE_BASE NUKE_SYNDICATE_BASE
 #define DETONATION_NEAR_MISSED_STATION NUKE_NEAR_MISS
 #define DETONATION_MISSED_STATION NUKE_MISS_STATION
+
+#define DEFAULT_NUKE_CODE "ADMIN"
```

Comment:
1. Can you rename it to something like `NUKE_CODE_UNSET` or something to be clear that this is an invalid code meant specifically for this purpose?
2. Auto doc it (triple slashes above the define)